FlowCrypt/Controllers/Settings/KeySettings/Key List/Helpers/KeySettingsProvider.swift

```diff
@@ -0,0 +1,57 @@
+//
+//  KeySettingsProvider.swift
+//  FlowCrypt
+//
+//  Created by Anton Kharchevskyi on 12/13/19.
+//  Copyright © 2019 FlowCrypt Limited. All rights reserved.
+//
+
+import Foundation
+
+protocol KeySettingsProviderType {
+    func getPublickKeys() -> Result<[KeySettingsItem], KeySettingsError>
+}
+
+enum KeySettingsError: Swift.Error {
+    case fetching, parsing
+}
+
+struct KeySettingsProvider: KeySettingsProviderType {
+    static var shared: KeySettingsProvider = KeySettingsProvider(
+        dataManager: DataManager.shared,
+        core: Core.shared
+    )
+
+    private let dataManager: DataManagerType
+    private let core: Core
+
+    func getPublickKeys() -> Result<[KeySettingsItem], KeySettingsError> {
```

Comment:
We do not expect any errors when parsing keys from our own storage - therefore this can be greatly simplified.

Reply:
Sure we do not expect this, nut what if smth went wrong. For example, keychain is blocked for this user. Or there was an error in key parsing. 
it should be handled in `error handling issue` that we already have 
https://github.com/FlowCrypt/flowcrypt-ios/issues/140